src/Middleware/SlimInitiator.php

```diff
@@ -103,6 +103,9 @@ protected function populateRoutes(RouteCollectorProxy $collector, array $routes)
             if ($details['file']) {
                 $route = $collector->get($details['route'], function(ServerRequestInterface $request, ResponseInterface $response) use ($details) {
                     $filename = GeneralUtility::getFileAbsFileName($details['file']);
+                    if ($details['contentType']) {
```

Comment:
I recommend using isset() so we avoid future PHP 8 issues :) Maybe, while you're at it, can you also fix the line 103 as well?

Reply:
Done. The general PHP8 thing I'd do in a separate PR.